test/meta_store_test.rb

```diff
@@ -15,7 +15,6 @@ def mock_request(uri, opts)
 
       def mock_response(status, headers, body)
         headers ||= {}
-        body = Array(body).compact
```

Comment:
Why did you remove this? Cleanup, or is it related to the problem with streamable bodies?